third_party/kiwi/kiwi/var.h

```diff
@@ -99,10 +125,35 @@ kiwi_vars_init(kiwi_vars_t *vars)
 	kiwi_var_init(&vars->vars[KIWI_VAR_STANDARD_CONFORMING_STRINGS],
 	              "standard_conforming_strings",
 	              28);
-	kiwi_var_init(
-	  &vars->vars[KIWI_VAR_APPLICATION_NAME], "application_name", 17);
+	kiwi_var_init_with_type(&vars->vars[KIWI_VAR_APPLICATION_NAME],
+	                        "application_name",
+	                        17,
+	                        KIWI_VAR_APPLICATION_NAME);
+	kiwi_var_init_with_type(&vars->vars[KIWI_VAR_SEARCH_PATH],
```

Comment:
do you actually need it with type? I think at this point client has no preferences about search path and app name..

Reply:
As I understand `od_deploy` is the main synchronization method between client configuration and current server connection. If we disconnect from the odyssey and try to connect with a new app name then in this method all values (from client and server) will be compared and if will find different than will build a query for change parameters on the server. Iteration goes through `kiwi_var_type_t` enum because need to add new var to this enum.

Reply:
Yes, but why not use kiwi_var_init()? As far as I can see you change application_name behavior and I want to understand if something is wrong currently.